hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/handlers/KeyEntityHandler.java

```diff
@@ -46,10 +49,39 @@ public KeyEntityHandler(
   @Override
   public NamespaceSummaryResponse getSummaryResponse()
           throws IOException {
-    NamespaceSummaryResponse namespaceSummaryResponse =
-            new NamespaceSummaryResponse(EntityType.KEY);
+    CountStats countStats = new CountStats(
+        -1, -1,
+        -1, 0);
+    return NamespaceSummaryResponse.newBuilder()
+        .setEntityType(EntityType.KEY)
+        .setCountStats(countStats)
+        .setObjectDBInfo(getKeyDbObjectInfo(getNames()))
+        .setStatus(ResponseStatus.OK)
+        .build();
 
-    return namespaceSummaryResponse;
+  }
+
+  private ObjectDBInfo getKeyDbObjectInfo(String[] names)
+      throws IOException {
+    OmKeyInfo keyInfo = getBucketHandler().getKeyInfo(names);
+    KeyObjectDBInfo keyObjectDBInfo = new KeyObjectDBInfo();
```

Comment:
Same as `BucketObjectDBInfo`, create constructor or builder which takes `OmKeyInfo` and create `KeyObjectDBInfo` object.